### PR TITLE
small javadoc improvements and corrections

### PR DESCRIPTION
**Pull request by PascalSchumacher**

### CODE REVIEW

File: src/main/java/org/assertj/core/api/AbstractIterableAssert.java [hunk 1]
```diff
@@ -106,10 +106,10 @@ public AbstractIterableAssert(ACTUAL actual, Class<?> selfType) {
   }
 
   /**
-   * Create a friendly soft or "hard" assertions.
+   * Create a friendly soft or "hard" assertion.
    * <p>
-   * Implementation needs to redefine it in order for some methods, such as {@link #extracting(Extractor)}, to be able
-   * to build appropriate list assert (eg: {@link ListAssert} versus {@link SoftAssertionListAssert}).
+   * Implementations needs to redefine it so that some methods, such as {@link #extracting(Extractor)}, are able
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/api/AbstractIterableAssert.java]: `needs` -> `need`

**Comment by joel-costigliola** [comment]
cool, just before the release planned today :)

**Comment by PascalSchumacher** [comment]
Release today?

Great news! :)

I

**Comment by joel-costigliola** [comment]
well maybe if you stop opening a zillion PR :smile_cat:

**Pull request merged**